lib/stanford-mods/date_parsing.rb

```diff
@@ -85,6 +86,55 @@ def self.facet_string_for_century(date_str)
         end
       end
 
+      # get String sortable value for B.C. if we have  B.C. pattern
+      #  note that these values must *lexically* sort to create a chronological sort.
+      #  We know our data does not contain B.C. dates older than 999, so we can make them
+      #  lexically sort by subtracting 1000.  So we get:
+      #    -700 for 300 B.C., -750 for 250 B.C., -800 for 200 B.C., -801 for 199 B.C.
+      # @param [String] date_str String containing B.C.
+      # @return [String, nil] String sortable -ddd if B.C. in pattern; nil otherwise
+      def self.sortable_year_for_bc(date_str)
```

Comment:
there's a PR coming with a method that ties all of these together.  The reason they are all class methods is so that other code (outside stanford-mods) can use these methods, once this is split off into a gem.